selfie.c

```diff
@@ -5631,6 +5626,14 @@ void selfie_compile() {
     emit_fetch_data_segment_size_interface();
   }
 
+  data_size = data_size + SIZEOFUINT64;
+
+  // emit global HOST_OS variable for bootstrapping
+  create_symbol_table_entry(GLOBAL_TABLE, string_copy("HOST_OS"), 1, VARIABLE, UINT64_T, SELFIE, -data_size);
```

Comment:
This is a problem since it introduces another global variable that shifts all offsets of all other global variables. How about modifying `identifier_or_keyword` to rewrite `HOSTOS` to `SELFIE`? Without `_` it's the same number of characters...

Reply:
I do not understand, why that would be a problem. We do this also with `_bump`. Or am I missing something?